app/services/blobs.rb

```diff
@@ -7,10 +7,17 @@ class Blobs
   def self.blobs_for(signed_ids)
     {}.tap do |blob_hash|
       signed_ids.each do |filename, signed_id|
+        next unless signed_id? signed_id
```

Comment:
Can/should we align this check with the recent change in the strategy used to determine if an ID is a signed ID or not? E.g.:

```suggestion
        next unless ActiveStorage.verifier.valid_message?(file_id)
```

And then we can ditch the new method below.